content/2017.03.building-projects-as-a-distributed-team.rst

```diff
@@ -0,0 +1,251 @@
+be focusing on for the next six months.
+
+This often means changing our course and killing projects or at least
+putting them on the side if team are turning over to new priorities.
+
+
+Our chance is opensource
+========================
+
+The chance we have at Mozilla, is that everything we do is published
+as Free, Libre, Opensource Software, which means that, even if, as
+Mozilla employee, we focus on something else, the work we've done is
+still available for the community to care about it.
+
+Thunderbird is a good example of this, Mozilla stopped spending
+resources on it in September 2007 but ten years later Mozilla
+Thunderbird is still actively used by a large number of users everyday.
+
+When building software we solve complex issues and all the solutions
+we have put together while doing so are really useful for other people
+that might need to solve similar problems.
+
+- Mozilla Persona lead the ground for Firefox Account and Portier.
+- The Firefox Hello loop server is a good example of how to build a
+  WebRTC signaling server.
+- The RemoteServiceWorker projects shows how to use Python 3 asyncio
+  to handle a large pool of websockets using redis to route messages
+  between them.
+
+
+Prototyping as a way to learn
+=============================
+  
+More than writting lines of code, we are researching solutions to
+complex problems at scale and turning this solution into, production
+ready, experiments.
+
+All this efforts are not lost because we learn a great deal from all
+of them.
+
+When we spend a month to one year building something, we are learning.
+
+But everything you learn is useful only if you share it.
+
+
+Sharing as a way to build software
+==================================
+
+In one sentence, what matters the most is to build software to put
+your vision in motion and to share what you've learned along the way.
+
+Luckily, there are people that you need to inform about what you are
+doing and learning about: your colleagues.
+
+You need to explain what you are trying to solve, you need to find out
+how to solve the issue and agree on the solution you are putting
+together, also having feedback will help you to choose the best
+solution even before having written a single line of code.
+
+
+The process of building software in a remote team
+=================================================
+
+Working in a remote team is really close from working with a
+community.
+
+You need to explicit everything you do, you need to find consensuses,
+you need to take feedback, you need to propose solutions and try
+things.
+
+As a team, as a community, you are stronger because instead of being
+alone you can rely on others to help you make the best choices and
+write the best software.
+
+
+The Minimal Viable Prototype (MVP)
+----------------------------------
+
+The first thing to do, is to gather together and brainstorm around the
+project to define what is the **Minimal Viable Prototype**.
```

Comment:
I think it's *Product*, not *Prototype*. I don't think a prototype can ever be "viable", or then it means shipping a prototype, which basically defeat its initial purpose of being a throwable experiment. But that may just be my interpretation.

Edit: not just mine apparently https://www.quora.com/What-is-the-difference-between-a-prototype-and-a-minimum-viable-product-MVP